cvat-sdk/cvat_sdk/core/uploading.py

```diff
@@ -374,7 +344,7 @@ def upload_files(
                 for i, filename in enumerate(group):
                     files[f"client_files[{i}]"] = (
                         os.fspath(filename),
-                        es.enter_context(closing(open(filename, "rb"))).read(),
+                        es.enter_context(open(filename, "rb")).read(),
```

Comment:
I think you can remove `.read()` too. There's not much point in holding the file open if you read its entire contents. Alternatively, the whole thing could be replaced with `filename.read_bytes()`.

Reply:
We can't just remove read, because file pointers turned out to be not supported by urllib (unlike requests). I've implemented the idea with `read_bytes`.